BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -944,6 +944,15 @@
 "SetUpUnlock" = "Set up unlock";
 "SetUpBiometricsOrChooseAPinCodeToQuicklyAccessYourVaultAndAutofillYourLogins" = "Set up biometrics or choose a PIN code to quickly access your vault and autofill your logins.";
 "SetUpLater" = "Set up later";
+"ChooseYourMasterPassword" = "Choose your master password";
+"ChooseYourMasterPasswordMessage" = "Choose a unique and strong password to  keep your information safe.";
+"MasterPasswordRequiredTitle" = "Master password (required)";
+"RetypeMasterPasswordTitle" = "Re-type master password (required)";
+"MasterPasswordHintTitle" = "Master password hint";
+"BitwardenCannotResetLostPasswordMessage" = "Bitwarden cannot reset a lost or forgotten master password.";
```

Comment:
🎨 I know there are a lot of old key/values that don't follow this but please keep the keys and values as aligned as possible unless the key would be very long. 
ℹ️ This helps with translation and how Crowdin is configured to detect changes, i.e. it will check whether the key has changed (not the value) in order to trigger new need for translations so by keeping the key/value aligned, if the value changes then the key would change as well which will trigger the need for new translations in Crowdin.
ℹ️ Additionally, given that we're sharing translations between the Android and iOS applications it's more likely that we use the exact same key between the apps if we keep the key/value aligned so less translations are duplicated.
ℹ️ Finally, it's easier when reading the code to see exactly what's being presented to the user.
```suggestion
"ChooseAUniqueAndStrongPasswordToKeepYourInformationSafe" = "Choose a unique and strong password to keep your information safe.";
"MasterPasswordRequired" = "Master password (required)";
"RetypeMasterPasswordRequired" = "Re-type master password (required)";
"MasterPasswordHint" = "Master password hint";
"BitwardenCannotResetALostOrForgottenMasterPassword" = "Bitwarden cannot reset a lost or forgotten master password.";
```